AutoDefineAddon/autodefine.py

```diff
@@ -253,181 +137,34 @@ def _get_definition(editor,
     if word == "":
         tooltip("AutoDefine: No text found in note fields.")
         return
-    valid_entries = get_preferred_valid_entries(editor, word)
 
     insert_queue = {}
 
+    cardBuilder = None
+    if settings.PREFERRED_DICTIONARY == "COLLEGIATE" or settings.PREFERRED_DICTIONARY == "MEDICAL":
+        cardBuilder = cardbuilder.CollegiateCardBuilder(word)
+    elif settings.PREFERRED_DICTIONARY == "SPANISH":
+        cardBuilder = cardbuilder.SpanishCardBuilder(word)
```

Comment:
let's add an `else` case here and `raise` or print an error message -- it looks like `validate_settings` may just allow continued execution